google/cloud/bigtable/tests/admin_backup_async_future_integration_test.cc

```diff
@@ -0,0 +1,279 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/bigtable/instance_admin.h"
+#include "google/cloud/bigtable/testing/table_integration_test.h"
+#include "google/cloud/internal/getenv.h"
+#include "google/cloud/internal/time_utils.h"
+#include "google/cloud/testing_util/assert_ok.h"
+#include "google/cloud/testing_util/chrono_literals.h"
+#include <google/protobuf/util/time_util.h>
+#include <gmock/gmock.h>
+
+namespace google {
+namespace cloud {
+namespace bigtable {
+inline namespace BIGTABLE_CLIENT_NS {
+namespace {
+namespace btadmin = google::bigtable::admin::v2;
+namespace bigtable = google::cloud::bigtable;
+
+class AdminBackupAsyncFutureIntegrationTest
+    : public bigtable::testing::TableIntegrationTest {
+ protected:
+  std::shared_ptr<AdminClient> admin_client_;
```

Comment:
clang-tidy says, "non-static data member 'admin_client_' of 'AdminBackupAsyncFutureIntegrationTest' shadows member inherited from type 'TableIntegrationTest'".